priv/perf/apps/load_test/lib/test_runner.ex

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.TestRunner do
+  @moduledoc """
+  This module runs tests using `mix run`. For example:
+
+  mix run -e "LoadTest.TestRunner.run()" -- "deposits" "1" "5"
+
+  It accepts three arguments:
+  - test name
+  - transactions per seconds
+  - period in seconds
+
+  Optional paramters:
+  - Percentile. You can pass percentile as the fourth parameter. By default `mean` value is used.
```

Comment:
where are these used???

Reply:
whats the purpose?

Reply:
It's used when checking error_rate. By default mean value is used. if percentile is set, error_rate percentile is used